art_archive_project/api/images/controllers/images_search.py

```diff
@@ -8,16 +7,16 @@
 from api.images.controllers import images_api
 from api.utils.json_decorator import json
 from api.utils.errors import bad_request, unprocessable_entry
+from api.utils.response_wrapper import ok_response
+from api.utils.url_helper import get_absolute_url
 
 
 @images_api.route("/search/", methods=['GET'])
 @json
 def images_search():
```

Comment:
Same goes for this function. There are multiple ways to divided this into a series of calls to helper functions. Let's try to slice it up.